Userland/Libraries/LibWeb/Layout/ListItemMarkerBox.cpp

```diff
@@ -21,6 +21,21 @@ ListItemMarkerBox::~ListItemMarkerBox()
 {
 }
 
+static const String number_to_alphabet(unsigned int number, const String alphabet)
+{
+    auto base = alphabet.length();
+    StringBuilder output_string;
+    while (number > base) {
+        number--;
+        auto remainder = number % base;
+        number -= remainder;
+        output_string.append(alphabet[remainder]);
+        number /= base;
+    };
+    output_string.append(alphabet[number - 1]);
+    return output_string.to_string().reverse();
+}
+
```

Comment:
This also exists in Spreadsheet (`String Spreadsheet::convert_to_string(size_t value, unsigned base = 26, StringView map = {})`) along with its inverse, maybe extract that and use it here?

Reply:
Ah shoot! Let's take this then. Should we move the functionality to String and friends somewhere?

Reply:
Yeah, probably `String::bijective_base_from(number, <base>, map)` or something?

Reply:
Sounds good.
I will do that ;)

Reply:
I'll merge this as-is, and we can follow up with changes to add a string helper for bijective basin'.